Terminal.Gui/Core/EscSeqUtils/EscSeqReq.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Terminal.Gui {
+	/// <summary>
+	/// Represents a state of escape sequence request.
+	/// </summary>
+	/// <remarks>
+	/// This is needed because there are some escape sequence requests responses that are equal
+	/// with some normal escape sequences and thus, will be only considered the responses to the
+	/// requests that were registered with this object.
+	/// </remarks>
+	public class EscSeqReqStat {
+		/// <summary>
+		/// Gets the terminating.
+		/// </summary>
+		public string Terminating { get; }
+		/// <summary>
+		/// Gets the number of requests.
+		/// </summary>
+		public int NumOfReq { get; }
+		/// <summary>
+		/// Gets information about unfinished requests.
+		/// </summary>
+		public int Unfinished { get; set; }
+
+		/// <summary>
+		/// Creates a new state of escape sequence request.
```

Comment:
AHHHH. It's supposed to be `EscSeqReqState` (with an `e`)! I think I get it now.

Better name might be `EscSequenceRequestState`.

Reply:
Done.